NodeJS/__integration__/AgentClient.it.ts

```diff
@@ -1,62 +1,81 @@
 import { AgentClient, createAgentClient } from "../src/AgentClient"
-import { RestClient } from "../src/RestClient"
+
+import { log4js } from "../src/Constants"
```

Comment:
what does this do? I'm not sure why you would want to be logging test stuffs.

Reply:
enable logging.  want to see what is going on during test.

Reply:
Also, what does this import gain us that normal `console.*` doesn't? In general and specifically in the tests.

Reply:
This enables logging inside tested class.